lib/spack/spack/util/environment.py

```diff
@@ -461,12 +486,50 @@ def is_unset(self, var_name):
         # unset
         return (type(var_updates[-1]) == UnsetEnv)
 
+    def blacklist(self, banned):
```

Comment:
I don't see this function used anywhere

Reply:
I think it was only used in a previous version of this. Should I remove it?

Reply:
IMO yes (generally I prefer that if a function is added that it is used, since otherwise concept drift can render it useless)